src/ml_tooling/plots.py

```diff
@@ -303,6 +303,9 @@ def plot_lift_curve(y_true, y_proba, title=None, ax=None):
 
     title = "Lift Curve" if title is None else title
 
+    # Get numpy array as _cum_gain_curve takes numpy arrays
+    y_true = getattr(y_true, 'values', y_true)
```

Comment:
I would probably just cast to a numpy array - that way we also support iterables and sequences

```suggestion
    y_true = np.array(y_true)
```